spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/configuration/TestDependencies.java

```diff
@@ -65,25 +68,30 @@
 @EnableHypermediaSupport(type = HAL)
 @Import(CompletionConfiguration.class)
 @EnableWebMvc
+@EnableConfigurationProperties(CommonApplicationProperties.class)
 public class TestDependencies extends WebMvcConfigurationSupport {
 
 	private final MavenProperties mavenProperties = new MavenProperties();
 
+	@Autowired
```

Comment:
Not sure what is the reasoning behind `autowiring` here. May be we just want to initialize the `CommonApplicationProperties` without having to do `@EnableConfigurationProperties`?